src/docs/product/accounts/quotas/index.mdx

```diff
@@ -208,7 +208,7 @@ If you have enabled the storage of crash reports, you may set limits for the max
 
 ## Inbound Filters {#inbound-data-filters}
 
-In some cases, the data you’re receiving in Sentry is hard to filter, or you don’t have the ability to update the SDK’s configuration to apply the filters. Sentry provides several methods to filter error events server-side, which apply before checking for potential rate limits. This does not apply to transaction events or attachments.
+In some cases, the data you’re receiving in Sentry is hard to filter, or you don’t have the ability to update the SDK’s configuration to apply the filters. Sentry provides several methods to filter error events server-side, which apply to all events before checking for potential rate limits.
```

Comment:
The updated text still specifically mentions error events, so I've made an update to fix that.

Also, based on what you brought up in the docs channel, thinking we should clarify that attachments aren't events wherever it makes sense.

```suggestion
In some cases, the data you’re receiving in Sentry is hard to filter, or you don’t have the ability to update the SDK’s configuration to apply the filters. Sentry provides several methods to filter all events and attachments server-side, which are applied before checking for potential rate limits.
```